src/components/carousel/README.md

```diff
@@ -117,6 +118,16 @@ Set the prop `indicators` to show the slide indicator buttons.
 
 Both indicators and controls can be set at the same time or independently.
 
+# Focus
+The carousel will focus the next slide for screen readers if not in play mode (doesn't have an interval)
+
+To avoid this, set the prop `focus` to false
```

Comment:
this is not recommended as it breaks ARIA and accessibility. 

It actually focuses the _current_ slide not the next one that isn't shown.